subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -259,6 +263,7 @@ static int net_key_status(struct bt_mesh_model *model,
 	struct net_key_param *param;
 	uint16_t net_idx;
 	uint8_t status;
+	int err = 0;
```

Comment:
Let's try to avoid initialisation upon declaration of variables when it's not necessary. This will help catch branches where the variable should have been set to a value (some error) but wasn't. Instead, add a `err = 0;` before the `done:` label. Same applies for all the functions you're modifying.

Reply:
ok，I see